src/app-layout/index.tsx

```diff
@@ -326,6 +326,9 @@ const OldAppLayout = React.forwardRef(
       onPreferencesChange: onSplitPanelPreferencesSet,
       reportSize: setSplitPanelReportedSize,
       reportHeaderHeight: setSplitPanelReportedHeaderHeight,
+      isStickyHeader: () => {
```

Comment:
`isStickyHeader` name means that there is a sticky header in app layout somewhere and split panel should adapt to that. In fact this is the split panel's header, so more correct name would be `headerShouldStick`